### PR TITLE
Added new release notes file 10.1 for review

### PR DESCRIPTION
**Pull request by SheilaBonnar**
@mgwelch @cmahond1 Added new release notes file 10.1 for review.

### CODE REVIEW

**Comment by mgwelch** [comment]
The main thing missing is breaking changes. I've asked @DMahoney0 to identify those in his CHANGELOG.

**Comment by DMahoney0** [comment]
For comparison purposes, here is my Changelog rough draft: https://github.com/metasys-server/api-docs/pull/42

Figured reviewers can indicate if they have other things they want to include from my PR and vice versa.

**Comment by mgwelch** [comment]
> For comparison purposes, here is my Changelog rough draft: #42

I assume this was already based on your PR. But I need you to call out breaking changes in the changelog

File: release_notes_edits_2.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - Get a single object attribute
+    - Update an object
+    - Get commands for an object
+    - Send a command to an object<br><br>
+- Schemas
+    - Get a single enumeration schema
+## Changes necessitating client refactor (breaking changes)
+ - Introduction 
+	- Added Base URL, this describes the base endpoint for the API.
+	- Removed Redirects. The client no longer handles inbound and outbound HTTP calls; a reverse proxy now handles this operation.<br><br>
```
Comment (mgwelch) [inline release_notes_edits_2.md]: Most likely this isn't a breaking change. A client should keep working if it was handling redirects properly.

**Comment by DMahoney0** [comment]
@SheilaBonnar, your commit now has 3 versions of the release notes file.  Are you going to merge these into one?

File: release_notes_edits.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Release notes 10.1
+
+## Added
+- Network devices
+    - Get network devices serving a space<br><br>
```
Comment (DMahoney0) [inline release_notes_edits.md]: This was new in 10.0: https://github.com/metasys-server/api-docs/pull/42/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR76

File: release_notes_edits.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - Send a command to an object<br><br>
+- Schemas
+    - Get a single enumeration schema
+## Changes necessitating client refactor (breaking changes)
+- Objects	
+	- Changed response payload 'enumSet' links to fully qualified enums.
+	- Changed 'attributes' link to 'trendedAttributesUrl'.
+## Improvements and optimizations
+- Introduction
+	- Added Base URL, this describes the base endpoint for the API.
```
Comment (DMahoney0) [inline release_notes_edits.md]: Extra white space before the hyphen

Comment (DMahoney0) [reply release_notes_edits.md]: @SheilaBonnar, this issue remains

**Comment by DMahoney0** [comment]
Redone in https://github.com/metasys-server/api-docs/pull/48
Closing.

**Pull request closed without merging**